### ISSUE TITLE
Use the new Hive logo?

### ISSUE THREAD

**Issue by weilu** [hivewallet/hive-mac#419]
![logo](https://hivewallet.com/assets/img/logo.png)

**Comment by jsuder** [comment]
@mattatgit do you have some kind of vector version of that last one with shadows?

**Comment by mattatgit** [comment]
@jsuder no, that one is built as a PSD, at 500px square. You specifically need vector for it ?

**Comment by jsuder** [comment]
Could be a PSD, just some kind of source file. I want to see if I can tweak it to a version that looks acceptable to me.

**Comment by mattatgit** [comment]
Ah i see. In that case, no, i don't. Sorry.

**Comment by jsuder** [comment]
Wait, you said you've built this icon as a PSD, can I get that PSD? :)

**Comment by mattatgit** [comment]
You can get a flat image no problem. A PSD so you can change it to suit yourself, no, that's not how this works. If you have a specific idea for an tweak, let me know. I don't ask for access to your code so i can  make it acceptable to me, so i expect that you return that courtesy.
